### ISSUE TITLE
Iterating over the process list while trying to find the game may randomly crash the application

### ISSUE THREAD

**Issue by soulweaver91** [Maselkov/GW2RPC#10]
GW2RPC crashed while I was sleeping, hours after the last time GW2 had been running. The log file showed this:

```
2020-12-04 07:01:48,671:CRITICAL:root: GW2RPC has crashed
Traceback (most recent call last):
  File "site-packages\psutil\_pswindows.py", line 636, in wrapper
  File "site-packages\psutil\_pswindows.py", line 702, in exe
ProcessLookupError: [Errno 3] No such process

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "gw2rpc\gw2rpc.py", line 373, in main_loop
  File "gw2rpc\gw2rpc.py", line 353, in update_gw2_process
  File "site-packages\psutil\__init__.py", line 661, in name
  File "site-packages\psutil\_pswindows.py", line 636, in wrapper
  File "site-packages\psutil\_pswindows.py", line 688, in name
  File "site-packages\psutil\_pswindows.py", line 641, in wrapper
psutil.NoSuchProcess: psutil.NoSuchProcess process no longer exists (pid=23148)
```

Looking at the relevant lines on this application's side, my guess is that one of the processes in the given process list exited right at the same time as GW2RPC was iterating over them. Just catching that exception somewhere as well might be enough if that is the case.

**Comment by Maselkov** [comment]
Seems like your observation is correct, will fix soon

**Comment by n1tr0-5urf3r** [comment]
Maybe fixed in dd80967dd66f50774cb2fac2c8470bf35ada0edf, needs some testing

**Comment by n1tr0-5urf3r** [comment]
Should be fixed with Version 2.2
